framework/decode/vulkan_enum_util.h

```diff
@@ -41,7 +41,7 @@ GFXRECON_BEGIN_NAMESPACE(enumutil)
  * @param value VkResult code to process.
  * @return String describing the specified VkResult code.
  */
-static const char* GetResultDescription(VkResult result)
+inline static const char* GetResultDescription(VkResult result)
```

Comment:
I think using `inline static` will generate a copy of `GetResultDescription()` in every compilation unit using the header, and those copies will be preserved through linking.
If we use plain `inline`, the linker is free to choose an arbitrary one of those copies and eliminate the others.